python/cdp/auth/test/test_jwt.py

```diff
@@ -73,6 +73,24 @@ def jwt_options():
     )
 
 
+@pytest.fixture
+def websocket_jwt_options():
+    """Fixture that provides JWT options for WebSocket testing with null request parameters.
+
+    Returns:
+        JwtOptions: JWT options for WebSocket testing
+
+    """
+    return JwtOptions(
+        api_key_id="test-key-id",
+        api_key_secret="dummy-secret",  # Will be replaced in tests
+        request_method=None,
+        request_host=None,
+        request_path=None,
+        expires_in=120,
+    )
```

Comment:
This seems duplicative, can we use [this](https://github.com/coinbase/cdp-sdk/pull/75/files#diff-23c1684f75fc13d15680e9c9a62e32fd4211681c5ba1bd4ac77c5bf1d9198c97R36)?